x/evm/module.go

```diff
@@ -44,6 +44,9 @@ var (
 	_ module.AppModuleBasic = AppModuleBasic{}
 )
 
+// ConsensusVersion defines the current module consensus version.
+const ConsensusVersion = 2
```

Comment:
Will we need to update this to 3?

Reply:
We bumped the consensus version for some param updates.  Version v0.14.0 upstream had version `1`, and now v0.21.0 upstream has version `5`.

It looks like:
 - v0.17.0 bumped to 2 and includes a migration no longer in v0.21.0.
 - v0.19.0 bumped to 3 and include a migration no longer in v0.21.0
 - v0.21.0 bumped to 5 and includes two migrations

We will have to port those old migrations over since we are jumping from v0.14.0 to v0.21.0 and this branch doesn't contain the migrations for 2 & 3.  In addition we already used consensus version 2, so either we need to update all the consensus version numbers by 1 OR delete our v2 migration on this branch and ensure we re-write the consensus version in the kava upgrade handler.